examples/demo-jobs/.neuro/batch-action.yml

```diff
@@ -12,7 +12,7 @@ outputs:
   needs: ["task_1_${{ inputs.arg1 }}_second", task_2]
   res1:
     descr: action result 1
-    value: ${{ to_json(needs) }}
+    value: ${{ fmt('task_1_{}_second', inputs.arg1) }}.needs.task1 task_2.needs.task2
```

Comment:
Do you want `res1` to be string with value `task_1_%arg1_value%_second.needs.task task_2.needs.task2`? I think something is still broken here, probaly is correct version is:
```
value: ${{ needs[fmt('task_1_{}_second', inputs.arg1)].task1 }} ${{ needs.task_2.task2 }}
```

Reply:
My bad. Please check the third version